examples/SetSparkClrPackageVersion.ps1

```diff
@@ -0,0 +1,102 @@
+#
```

Comment:
As discussion we can keep the version identifiers updated prior to release tagging